project.clj

```diff
@@ -39,6 +39,7 @@
                                org.apache.httpcomponents/httpclient
                                net.sourceforge.nekohtml/nekohtml
                                ring/ring-core]]
+                 [com.cemerick/url "0.1.1"]                           ; URL library
```

Comment:
That's a big with that then. I'll fix it. But we use the site URL in lots of places and have never needed a new library to do it